protos/flyteidl/admin/node_execution.proto

```diff
@@ -211,6 +211,10 @@ message NodeExecutionGetDataResponse {
     // Full_outputs will only be populated if they are under a configured size threshold. 
     core.LiteralMap full_outputs               = 4;
 
+    // String location uniquely identifying where the deck HTML file is
+    // Should start with the storage location (e.g. s3://, gs:// etc.)
+    string deck_uri                            = 5;
```

Comment:
How will this be used? is there a design doc (1-pager) I missed? should this be a pre-signed url instead? 
If we have a design doc, do you mind linking it in the PR? if not, do you mind writing a quick 1-pager of how this all fits together?

Reply:
It's a pre-signed url here. 
https://docs.google.com/document/d/1hIzKEZuk3EQux0R5OJDAYUQmqMF1bbNz7-fAHl1vwH0/edit

TLDR, flyteconsole will retrieve `deck_uri` (pr-signed URL) from `NodeExecutionGetDataResponse`, and then use ifrmae to display the deck.
Something like, 
```
<iframe src="s3://bucket/deck.html">
```

Reply:
@EngHabu mind taking a look